Desktop/property-pro-app/src/components/Midsection/Midsection.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import "./midsection.css";
+
+export default function Midsection() {
+  return (
+    <div className="mid">
+      <div className="boxcontainer">
+        <div className="buy">
+          <div className="icon">
+            <img
+              src="https://img.icons8.com/carbon-copy/100/000000/home.png"
+              alt="house.img"
+            />
+          </div>
+          <div className="middleheader">
+            <p className="caption">
+              {" "}
+              <b> BUY A HOME </b>{" "}
+            </p>
+            <p>
+              {" "}
+              Find your place with an immersive photo and top listing, including
+              things you won't find anywhere else.
+            </p>
+          </div>
+          <div className=" search">
+            <button className="searchbutton">Search Home </button>
+          </div>
+        </div>
+      </div>
+
+      <div className="boxcontainer">
+        <div className="buy">
+          <div className="icon">
+            {" "}
+            <img
+              src="https://img.icons8.com/carbon-copy/100/000000/home.png"
+              alt="house.img"
```

Comment:
house.img is not good enough for alternative name. Your app user won't know what house.img means if the image didn't come up, so i suggest you make it simple just use "home".